CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 3.10)
 
 project(obs-vkcapture VERSION 1.0.1)
 
+set(SOCKNAME "/com/obsproject/vkcapture" CACHE STRING "Socket name")
+
+add_definitions("-DSOCKNAME=(\"${SOCKNAME}\")")
```

Comment:
I don't really like it being configurable. Any usecase for this?

Reply:
Isn't the reason it doesn't support multiple windows that a single socket is used?

Reply:
Ah, right, it's a stream socket and works with `connect()`.

Would you like me to hardcode the socket name?

Reply:
No need for `strlen()`, we can use `sizeof()`.